lib/util/stream_utils.js

```diff
@@ -219,6 +219,7 @@ shaka.util.StreamUtils.getVariantTracks =
     function(period, activeAudioId, activeVideoId) {
   var StreamUtils = shaka.util.StreamUtils;
   var variants = StreamUtils.getPlayableVariants(period.variants);
+  var label;
```

Comment:
Initialize this to null, since the type allows for null instead of undefined.  A minor difference, I know.